sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md

```diff
@@ -7,7 +7,7 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://github.com/Azure/azure-rest-api-specs/blob/40008a6fae4c614cfed5f401a635462eb4e93a93/specification/cognitiveservices/data-plane/FormRecognizer/stable/v2.0/FormRecognizer.json
+    -  https://github.com/Azure/azure-rest-api-specs/blob/8fee11f5dd30e228a850ee25a2f3368dc1671b53/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json
```

Comment:
should this be referencing the swagger in master?

Reply:
Our CI always generates the code looking at this file and makes a comparison between generated code and what the generated code in the branch looks like. If they don't match the build breaks.
We avoid pointing to master because if the service makes an update to the swagger, it could break us so it could break the build.
Also, our CI checks that we don't point to master

Reply:
Got it, thanks

Reply:
Sure! curious, how does it work on Python land?

Reply:
Hmm our CI does not generate the code automatically for us. We run autorest locally on the target swagger file and then manually update the generated code.